client/me/purchases/manage-purchase/change-payment-method/index.jsx

```diff
@@ -112,24 +120,23 @@ function ChangePaymentMethod( props ) {
 
 			<Layout>
 				<Column type="main">
-					<StripeHookProvider
-						locale={ props.locale }
-						configurationArgs={ { needs_intent: true } }
-						fetchStripeConfiguration={ getStripeConfiguration }
-					>
-						{ isEnabled( 'purchases/new-payment-methods' ) ? (
-							<ChangePaymentMethodList />
-						) : (
-							<PaymentMethodForm
-								apiParams={ { purchaseId: props.purchase.id } }
-								initialValues={ props.card }
-								purchase={ props.purchase }
-								recordFormSubmitEvent={ recordFormSubmitEvent }
-								siteSlug={ props.siteSlug }
-								successCallback={ successCallback }
-							/>
-						) }
-					</StripeHookProvider>
+					{ isEnabled( 'purchases/new-payment-methods' ) ? (
+						<ChangePaymentMethodList
+							currentPaymentMethod={ props.card }
+							purchase={ props.purchase }
+							isDataLoading={ isDataLoading }
```

Comment:
I think we can remove `isDataLoading` here and in `ChangePaymentMethodList` because with the early return above it will always be `false` when we get here.